WcaOnRails/app/assets/javascripts/persons.js

```diff
@@ -15,11 +15,6 @@ onPage('persons#index', function() {
   var $table = $('.persons-table');
   var options = $table.bootstrapTable('getOptions');
 
-  // Set the table options from the url params.
-  options.pageNumber = parseInt($.getUrlParam('page')) || options.pageNumber;
```

Comment:
I believe this is necessary for links like this: https://www.worldcubeassociation.org/persons?page=2&region=all&search= (note the `page=2` parameter in there). It may look like this `options` variable is unused, but I'm pretty sure if you dig, you'll see that the call to `'getOptions'` above actually returns a reference to an object internal to the bootstrap table library that we can mutate. That's what the comment "Load the data using the options set above." is referring to. @jonatanklosko implemented this in https://github.com/thewca/worldcubeassociation.org/pull/736.

Here's what I see when I load https://www.worldcubeassociation.org/persons?page=1&region=all&search=:

![image](https://cloud.githubusercontent.com/assets/277474/21748088/028af38c-d530-11e6-9f4f-4d44339ba443.png)

I'm guessing you're trying to get rid of this first, cancelled ajax request. I added a breakpoint to figure out where this is coming from, and here's the javascript backtrace:

![image](https://cloud.githubusercontent.com/assets/277474/21748136/aadc8c0c-d531-11e6-9cc8-0cd758c03620.png)

It looks like the only way to get this initial request to stop is to get data to be equal to false [here](https://github.com/wenzhixin/bootstrap-table/blob/60a8c3bcd50ab4821d805c5478024082e3147e0a/src/bootstrap-table.js#L1980-L1983).

Probably, the only solution is to correctly set the `data-page-number` option before the bootstrap table plugin is intialized on the table ([here](https://github.com/jfly/worldcubeassociation.org/blob/22de696bf09f01d090298436f817e0e829861479/WcaOnRails/app/assets/javascripts/application.js#L359)), rather than trying to set it *after* the table is created. I'm not sure what a clean solution to that would look like, though.